config/elasticlens.php

```diff
@@ -26,4 +26,18 @@
     'index_paths' => [
         'app/Models/Indexes/' => 'App\Models\Indexes',
     ],
+
+    'chunk_rates' => [
+        'default' => 1000,
+        'migrate_default' => 100,
+
+        'relationship_scaling' => [
+            'enabled' => true,
+            'thresholds' => [
+                3 => 750,
+                6 => 500,
+                9 => 250,
+            ],
+        ]
```

Comment:
I also thought about adding a configurable model-dependent threshold.
But that might just complicate things.